installer.sh

```diff
@@ -46,6 +47,23 @@ if snap list docker >/dev/null 2>&1; then
     DOCKER_SERVICE="snap.docker.dockerd.service"
 fi
 
+# Detect wrong docker logger config
+if [ ! -f "$DOCKER_DAEMON_CONFIG" ]; then
+  # Write default configuration
+  cat > "$DOCKER_DAEMON_CONFIG" <<- EOF
+    {
+        "log-driver": "journald"
+    }
+EOF
+  # Restart Docker service
+  systemctl restart "$DOCKER_SERVICE"
```

Comment:
This might be problematic when re-running the installation script? As it will shut down all containers, including the running Supervisor.

Reply:
Only if the daemon config does not exist.
Rerunning the script will restart the supervisor anyway :man_shrugging:

https://github.com/home-assistant/supervised-installer/blob/756f6db062aa1f315ac04d7831cb255b1c683445/installer.sh#L182
https://raw.githubusercontent.com/home-assistant/supervised-installer/master/files/hassio-supervisor.service

Reply:
Should be fine, let's also add fix the storage driver